src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (x) => {
+    if (x === undefined) {
```

Comment:
```suggestion
    if (!x) {
```

or maybe even use && operator for this